client/core/trade.go

```diff
@@ -741,15 +746,29 @@ func (t *trackedTrade) isSwappable(match *matchTracker) bool {
 		return false
 	}
 
-	if dbMatch.Side == order.Taker && metaData.Status == order.MakerSwapCast {
-		// Check the confirmations on the maker's swap.
-		confs, req, changed := t.counterPartyConfirms(match)
-		ready := confs >= req
-		if changed && !ready {
-			t.dc.log.Debugf("Match %v not yet swappable: current confs = %d, required confs = %d",
-				match.id, confs, req)
+	if metaData.Status == order.MakerSwapCast {
+		// Get the confirmation count on the maker's coin.
+		if dbMatch.Side == order.Taker {
+			// If the maker is the counterparty, we can determine swappability
+			// based on the confirmations.
+			confs, req, changed := t.counterPartyConfirms(match)
+			ready := confs >= req
+			if changed && !ready {
+				t.dc.log.Debugf("Match %v not yet swappable: current confs = %d, required confs = %d",
+					match.id, confs, req)
+			}
+			return ready
+		} else {
```

Comment:
No `else` needed since previous block ends with a `return`.  Same in `isRedeemable`.